train/augment_dataset.py

```
@@ -0,0 +1,505 @@
+    "runtime. (e.g. 'play {song} by {artist}').\n"
+    "When you see them ALWAYS replace the placeholder with a diverse, realistic example "
+    "value (e.g. 'play bohemian rhapsody by queen', 'play hotel california by eagles'). "
+    "Use varied, realistic values across all generated utterances — "
+    "different names, places, numbers, genres, etc. — never the same value twice."
+)
+
+
+def build_user_prompt(lang: str, domain: str, intent: str,
+                      examples: list[str], n: int) -> str:
+    """Build the user-turn prompt for a single augmentation request."""
+    intent_display = intent.replace(".", " ").replace("_", " ").rstrip(" intent")
```

Ruff (0.15.2)

[warning] Using `.strip()` with multi-character strings is misleading

(B005)